scripts/microshift-sos-report.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPT_NAME=$(basename "$0")
+PROFILES="network,security,microshift"
+PLUGINS="container_log,crio,rpmostree,rpm"
+TEMPDIR="/tmp"
+
+function usage() {
+    echo "Collect an sos report including all relevant information for MicroShift"
+    echo "Includes"
+    echo "  profiles: ${PROFILES}"
+    echo "  plugins: ${PLUGINS}"
+    echo ""
+    echo "Usage: ${SCRIPT_NAME} [--tmp-dir TMP-DIR]"
+    echo "   --tmp-dir TMP-DIR  Temporary directory where report will be written. Defaults to ${TEMPDIR}."
+    exit 1
+}
+
+if [ $# -ge 1 ]; then
+    case $1 in
+    --tmp-dir)
+        TEMPDIR="$2"
```

Comment:
This would generate uninitialized variable error if run with one argument.
I think we need to change the "if" to be `$# -eq 2` else `usage`

Reply:
Done

Reply:
Sorry, for the incomplete suggestion. I guess, we need to check for 0 or 2 arguments to the script because `--tmp-dir` is optional, right?

Reply:
Done